pkg/app/app.go

```diff
@@ -46,10 +46,7 @@ func NewApp(config config.AppConfigurer) (*App, error) {
 	}
 	var err error
 	app.Log = newLogger(config)
-	app.OSCommand, err = commands.NewOSCommand(app.Log)
-	if err != nil {
-		return app, err
-	}
+	app.OSCommand = commands.NewOSCommand(app.Log)
```

Comment:
I want to keep the error check here even if currently it's impossible for NewOSCommand to return an error, because down the line the method might have more complexity, and may need to return an error

Reply:
Yep but in my opinion, if you want to do this later, simply refactor and add the error at this moment, specially with compiling it's easy to catch every point where you have to change. Generally speaking, you don't have to add code "just in case", it adds dead code and uneeded burden.

Reply:
I wholeheartedly agree!